editor/src/com/talosvfx/talos/editor/project2/GlobalDragAndDrop.java

```diff
@@ -71,10 +73,25 @@ public void dragStop (InputEvent event, float x, float y, int pointer, DragAndDr
 	}
 
 	public void fakeDragDrop (int screenX, int screenY, String[] absoluteFiles) {
+		if (absoluteFiles.length == 1) {
+			FileHandle handle = Gdx.files.absolute(absoluteFiles[0]);
+			if (handle.extension().equals(TalosProjectData.TALOS_PROJECT_EXTENSION)) {
+				// load the project
+				TalosProjectData data = TalosProjectData.loadFromFile(handle);
+				SharedResources.projectLoader.loadProject(data);
```

Comment:
Im not sure if we ever want this, and if we do it definitely needs to be done from some external dialog before GloballDragAndDrop system.

Reply:
Fixed!